src/Nethermind/Nethermind.Runner/configs/joc-testnet.cfg

```diff
@@ -0,0 +1,25 @@
+{
+    "Init": {
+        "ChainSpecPath": "chainspec/joc-testnet.json",
+        "GenesisHash": "0x0fb7b4779aae36dc557227283f182bc9a3b232c07fae4a2553734c5817df1d06",
+        "BaseDbPath": "nethermind_db/joc-testnet",
+        "LogFileName": "joc-testnet.logs.txt"
+    },
+    "Sync": {
+        "FastSync": true,
+        "SnapSync": true,
+        "FastBlocks": true
+    },
+    "Metrics": {
+        "NodeName": "Joc"
+    },
+    "Blocks": {
+        "TargetBlockGasLimit": 30000000
```

Comment:
are you sure it is correct for JOC?

Reply:
Looking at the JOC block explorer
https://explorer.japanopenchain.org/block/8453482/transactions

gas limit is 30mln